apps/golden-sample-app/src/app/app.module.ts

```diff
@@ -97,32 +97,34 @@ import { ActivityMonitorModule } from './auth/activity-monitor';
       },
     },
     { provide: OAuthStorage, useFactory: () => localStorage },
-    {
-      provide: APP_INITIALIZER,
-      multi: true,
-      deps: [
-        OAuthService,
-        CookieService,
-        AuthEventsHandlerService,
-        AuthService,
-      ],
-      useFactory:
-        (oAuthService: OAuthService, cookieService: CookieService) =>
-        async () => {
-          // Remove this if auth cookie is not needed for the app
-          oAuthService.events.subscribe(({ type }) => {
-            if (type === 'token_received' || type === 'token_refreshed') {
-              // Set the cookie on the app domain
-              cookieService.set(
-                'Authorization',
-                `Bearer ${oAuthService.getAccessToken()}`
-              );
-            }
-          });
+    environment.mockEnabled
+      ? []
```

Comment:
should it be an empty object?

Reply:
Provider accepts 

`export declare type Provider = TypeProvider | ValueProvider | ClassProvider | ConstructorProvider | ExistingProvider | FactoryProvider | any[];`

If you wish to add any other type apart from a provider then it should be an array